packages/cli/src/services/orchestration/main/MultiMainSetup.ee.ts

```diff
@@ -78,7 +78,7 @@ export class MultiMainSetup extends EventEmitter {
 			config.set('multiMainSetup.instanceType', 'follower');
 
 			/**
-			 * Lost leadership - stop triggers, pollers, pruning, wait tracking, license renewal
+			 * Lost leadership - stop triggers, pollers, pruning, wait tracking, license renewal, queue recovery
 			 */
 			this.emit('leader-stepdown');
```

Comment:
Would it make sense to convert this to a `constant` so everywhere that references it becomes easy to find?

Reply:
(to prevent having to update the list in a comment forever)

Reply:
As discussed this will be simplified once we follow the pattern from https://github.com/n8n-io/n8n/pull/9724